nest/00. Foreword.md

```diff
@@ -0,0 +1,21 @@
+# Getting started with Nest
+## Foreword
+Welcome to Nest's official getting-started guide! This guide assumes you are using the latest version of Nest, which is 0.1.0 at the time of writing.
+
+In order to avoid any confusion, we will refer to the project as a whole as `Nest` (with an upper case n), while we will refer to the binary as `nest` (with a lower case n). Thus, `finest` refer to its eponymous binary, and is included in Nest.
+
+[//]: # (TODO: add link to the installation instructions page of Raven-OS)
+Nest is a dependency-based package manager for [Raven-OS](https://raven-os.org), which is an adaptive, customizable and evolutive open-source Operating System (OS) based on the Linux kernel. To use Nest, it's recommended to install Raven-OS to run Nest in it, because Nest may overlap with an already installed package manager. To install Raven-OS, follow the instructions [here](). Nest is included in the installation of Raven-OS by default, so you can start using it right away.
```

Comment:
```suggestion
Nest is a dependency-based package manager for [Raven-OS](https://raven-os.org), which is an adaptive, customizable and evolutive open-source Operating System (OS) based on the Linux kernel. To use Nest, it's recommended to install Raven-OS and to run Nest in it, because Nest may overlap with an already installed package manager. To install Raven-OS, follow the instructions [here](). Nest is included in the installation of Raven-OS by default, so you can start using it right away.
```